src/screens/ForgotPassword.js

```diff
@@ -1,105 +1,98 @@
-import { Pressable, StyleSheet, Text, TextInput, View } from 'react-native'
+import { Pressable, StyleSheet, Text, TextInput, View, SafeAreaView, Dimensions } from 'react-native'
 import React, { useState } from 'react'
+import Icon from 'react-native-vector-icons/FontAwesome';
 
-const ForgotPassword = () => {
-
+const { width, height } = Dimensions.get('window');
 
+const ForgotPassword = ({ navigation }) => {
   const [mail, setMail] = useState("")
 
-  const HandleButton =() =>{
-     console.log(mail)
-
+  const HandleButton = () => {
+    console.log(mail)
```

Comment:
```suggestion
  const HandleButton = () => {
    console.log(mail)
```

Görev kartında belirtildiği gibi bu bölüm navigation kullanılarak "LoginPage" sayfasına yönlendirilmesi gerekiyor.